django_rq/workers.py

```diff
@@ -2,24 +2,42 @@
 from rq.utils import import_attribute
 
 from .queues import get_queues
-from .settings import EXCEPTION_HANDLERS
 
 
-def get_exception_handlers():
-    """
-    Custom exception handlers could be defined in settings.py:
-    RQ = {
-        'EXCEPTION_HANDLERS': ['path.to.handler'],
+def get_exception_handlers(exception_handlers):
+    """Custom exception handler defined in QUEUE settings:
+    RQ_QUEUES = {
+        'default': {
+            'HOST': 'localhost',
+            'PORT': 6379,
+            'DB': 0,
+            'PASSWORD': '',
+            'DEFAULT_TIMEOUT': 360,
+            'EXCEPTION_HANDLERS': [
+                'test_exception_handler.my_custom_exception'
+            ],
+        }
     }
     """
-    return [import_attribute(path) for path in EXCEPTION_HANDLERS]
+    try:
+        from django.settings import RQ_EXCEPTION_HANDLERS
```

Comment:
`RQ_EXCEPTION_HANDLERS` should only be a fallback. So we only try to import it if `exception_handlers` is `None`.